libr/debug/p/native/w32.c

```diff
@@ -615,4 +615,29 @@ RList *w32_pids (int pid, RList *list) {
 	return list;
 }
 
+int w32_terminate_process (RDebug *dbg, int pid) {
+	HANDLE process = dbg->process_handle;
+	if (process == INVALID_HANDLE_VALUE) {
+		return R_FALSE;
+	}
+
+	/* stop debugging if we are still attached */
+	DebugActiveProcessStop (pid);
+	if (TerminateProcess (process, 1) == 0) {
+		print_lasterr ((char *)__FUNCTION__, "TerminateProcess");
+		return R_FALSE;
+
+	}
+	DWORD ret_wait;
+	/* wait up to one second to give the process some time to exit */
+	if ((ret_wait = WaitForSingleObject (process, 1000)) != 0) {
```

Comment:
Good catch. There is actually no "success" return return value like 0. I will change it.